starfinder/db/models.py

```diff
@@ -381,21 +512,37 @@ class RaceAssociatedFeat(db_engine.Model, ModelBase, HasId):
     feat_id = db_engine.Column(db_engine.ForeignKey("feats.id"),
                                      nullable=False)
 
+class FeatOption(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
+    feat_id = db_engine.Column(db_engine.ForeignKey("feats.id"),
+                                     nullable=False)
+    parent_id = db_engine.Column(db_engine.Integer(), nullable=False)
+    name = db_engine.Column(db_engine.String(64), nullable=False)
+    level = db_engine.Column(db_engine.Integer(), nullable=False)
+    description = db_engine.Column(db_engine.String(64), nullable=False)
+
+
+
+
```

Comment:
can you please go back to ensure that there is exactly two spaces below every class?

This is different from the dividers (ie. `L529`), which should only have one space below them.